### ISSUE TITLE
Code fix for CS8345 to make struct a ref struct

### ISSUE THREAD

**Issue by Neme12** [dotnet/roslyn#31831]
```c#
using System;

struct S
{
    Span<int> span; // CS8345: Field or auto-implemented property cannot be of type 'Span<int>' unless it is an instance member of a ref struct.
}
```

It would be nice to have a code fix to make `S` a ref struct.

**Comment by CyrusNajmabadi** [comment]
@Neme12 This is something that would likely be faster to just fix by opening a PR.  Would you be willing to do that?

**Comment by MaStr11** [comment]
Related #23326

**Comment by jcouv** [comment]
Fix was merged to 16.1-preview1
